checkipaconsistency/freeipaserver.py

```diff
@@ -138,6 +139,10 @@ def _search(self, base, fltr, attrs=None, scope=ldap.SCOPE_SUBTREE):
         except (ldap.NO_SUCH_OBJECT, ldap.SERVER_DOWN) as e:
             self._log.debug(self._get_ldap_msg(e))
             results = False
+        except ldap.REFERRAL as e:
+            self._log.critical("Replica %s is being re-initialized" % self._fqdn)
```

Comment:
389-ds can set a backend to referral for a variety of reasons: This message may be clearer if you say "Replica is temporarily unavailable: referral direct us to <list of hosts in referral error>". An example of this is online db2index or online full reinit will both set the mapping tree entry to referral instead of backend.